### ISSUE TITLE
@View.templateUrl should allow URL relative to the component (alpha-36)

### ISSUE THREAD

**Issue by wardbell** [angular/angular#4039]
In alpha-36, the `templateUrl` is relative to the application root url.

```
@View({
  templateUrl: 'app/hero-detail-component.html' // relative to the application root
})
```

That is fine if everyone is keeping their templates in a template directory. But if you like your template next to the component, you want to specify a url relative to the component.

That would become possible if allowed a url syntax with leading '.'. 

In this next example, the template is a sibling of the component class file:
```
@View({
  templateUrl: './hero-detail-component.html' // sibling of the component file
})
```
**This might be appropriate for `styleUrls` too**

@johnpapa agrees.

**Comment by gdi2290** [comment]
I think you're looking for https://github.com/angular/angular/issues/2383

**Comment by wardbell** [comment]
@gdi2290 Yup. That's it. Sorry for duplication. Maybe this helps because that issue is freighted with discussion that is a bit hard to follow.

**Comment by mhevery** [comment]
Closing as dup of #2383
